adhocracy4/polls/static/PollDetail/PollQuestion.jsx

```diff
@@ -48,11 +48,12 @@ export const PollQuestion = (props) => {
   const handleMultiChange = (event, isOther) => {
     const choiceId = parseInt(event.target.value)
     setUserChoices(prevState => [...prevState, choiceId])
+    setOtherChoiceAnswer(otherChoiceAnswer)
     props.onMultiChange(props.question.id, choiceId)
-    if (!isOther) {
-      props.onOtherChange(props.question.id, '', findOtherChoice())
-    } else {
+    if (isOther) {
       setOtherChoiceAnswer('')
+    } else {
+      props.onOtherChange(props.question.id, otherChoiceAnswer, findOtherChoice())
```

Comment:
should this be inside another if? although the initial condition should stop anything being saved that shouldn't be, right?

Reply:
I can't remember :(